### ISSUE TITLE
Training & resources

### ISSUE THREAD

**Issue by ADubhlaoich** [ADubhlaoich/accessible_accessibility#3]
There's quite a lot of stuff online that does its best to explain the aims of the WAI and what each bit of WCAG means. I'm gonna dump as much stuff as I can in here for later reference - I already put together some stuff to pin on the ITC Slack for people to browse through.

Most notably, I've gotten access to [deque University](https://dequeuniversity.com/) as of about 10 minutes ago.

**Comment by ADubhlaoich** [comment]
There's [Open Educational Resources](https://moocap.gpii.eu/?page_id=839) which came from MOOCAP, that DIT participated in, [Web Accessibility for Developers](https://www.canvas.net/browse/ryersonu/courses/adv-web-accessibility), and [Accessibility](https://developer.mozilla.org/en-US/docs/Learn/Accessibility) on MDN.

Google also had a standalone course I went through once, but it was atrocious and appeared to be hosted on some bespoke framework in a hidden URL... which may explain why I can't find it anymore.
